cmd/celestia-appd/cmd/root.go

```diff
@@ -44,6 +45,8 @@ const (
 
 	// FlagLogToFile specifies whether to log to file or not.
 	FlagLogToFile = "log-to-file"
+
+	UpgradeHeightFlag = "upgrade-height"
```

Comment:
[question][not blocking] I wonder if this flag should explicitly state `v2` so that users don't get confused by this flag for future upgrades. If it doesn't explicitly state `v2`, I think a godoc that explains it's only used for `v2` would be helpful.

Reply:
Yeah I think that's a good idea. So `v2-upgrade-height`?